dist/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+
```

Comment:
The pattern you're missing here is `.*` which matches any _hidden_ files that might end up in here.